README.md

```diff
@@ -222,17 +222,17 @@ Check out the [contributor guide](https://github.com/onnx/onnx/blob/master/docs/
 
 
 <!-- links -->
-[overview]: https://github.com/onnx/onnx/blob/master/docs/Overview.md
-[ir]: https://github.com/onnx/onnx/blob/master/docs/IR.md
-[versioning]: https://github.com/onnx/onnx/blob/master/docs/Versioning.md
-[operators]: https://github.com/onnx/onnx/blob/master/docs/Operators.md
-[python_api]: https://github.com/onnx/onnx/blob/master/docs/PythonAPIOverview.md
-[shape_inference]: https://github.com/onnx/onnx/blob/master/docs/ShapeInference.md
-[version_converter]: https://github.com/onnx/onnx/blob/master/docs/VersionConverter.md
-[new_op]: https://github.com/onnx/onnx/blob/master/docs/AddNewOp.md
-[community]: https://github.com/onnx/onnx/tree/master/community
-[sigs]: https://github.com/onnx/onnx/tree/master/community/sigs.md
-[wgs]: https://github.com/onnx/onnx/tree/master/community/working-groups.md
-[contributing]: https://github.com/onnx/onnx/blob/master/docs/CONTRIBUTING.md
-[CMakeLists]: https://github.com/onnx/onnx/blob/master/CMakeLists.txt
-[license]: https://github.com/onnx/onnx/blob/master/LICENSE
\ No newline at end of file
+[overview]: docs/Overview.md
+[ir]:   docs/IR.md
+[versioning]: docs/Versioning.md
+[operators]: docs/Operators.md
+[python_api]: docs/PythonAPIOverview.md
+[shape_inference]: docs/ShapeInference.md
+[version_converter]: docs/VersionConverter.md
+[new_op]: docs/AddNewOp.md
+[community]: community
+[sigs]: community/sigs.md
+[wgs]: community/working-groups.md
+[contributing]: docs/CONTRIBUTING.md
+[CMakeLists]: CMakeLists.txt
+[license]: LICENSE
```

Comment:
https://github.com/onnx/onnx/pull/3376 ONNX recently updated these URLs as absolute links for PyPI description (https://pypi.org/project/onnx/) to access it as well

Reply:
Thanks, good catch!

Reply:
Is there some way of meeting both goals?

Reply:
I think we might need more code to achieve (like string convert here https://github.com/onnx/onnx/blob/e3900651f056d559a72211c86f561f70d4c9f2e9/setup.py#L341), which would be quite complicated.